### PR TITLE
Fixed the volumetric clouds offset value bing clamped (case 1357318).

### PR DESCRIPTION
**Pull request by anisunity**
https://fogbugz.unity3d.com/f/cases/1357318/
https://fogbugz.unity3d.com/f/cases/1354920/
This PR removes the clamping on the shape offsets and adds two new parameters on the volume component to control the vertical speed of the shaping and erosion.
updated the doc

Testing status
Pair tested with @pierre-unity
For shape offset:
- made sure that all integer values return the same result
- made sure that ie behaved correctly while moving the slider
For vertical wind speed:
- made sure that the winds were in the correct orientation
- made sure that the winds affected correctly the clouds
- made sure that negative values worked as expected.

### CODE REVIEW

**Review by iM0ve — APPROVED** [review]
The PR is simple enough, I additionally checked that there ware no warnings and undo works.

**Demo:**

https://user-images.githubusercontent.com/36502659/129195940-b2e69072-4058-4f0d-9e06-f1c0ae3c562f.mp4

**Review by JulienIgnace-Unity — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/CHANGELOG.md [hunk 1]
```diff
@@ -360,6 +360,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed Additional Velocity for Alembic not taking correctly into account vertex animation
 - Fixed wrong LUT initialization in Wireframe mode.
 - Support undo of HDRP Global Settings asset assignation (case 13429870).
+- Fixed the volumetric clouds offset value bing clamped (case 1357318).
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/CHANGELOG.md:363]: Fixed an issue that clamped the volumetric clouds offset value (case 1357318).

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Small typo in the changelog to fix, otherwise good!

File: com.unity.render-pipelines.high-definition/Documentation~/Override-Volumetric-Clouds.md [hunk 2]
```diff
@@ -107,11 +107,13 @@ When importing these two map textures, disable **sRGB**. For best results, do no
 
 | **Property**                     | **Description**                                              |
 | -------------------------------- | ------------------------------------------------------------ |
-| **Global Wind Speed**            | Sets the global wind speed in kilometers per hour.<br />This value can be relative to the **Global Wind Speed** defined in the **Visual Environment**. |
+| **Global HorizontalWind Speed**  | Sets the global horizontal wind speed in kilometers per hour.<br />This value can be relative to the **Global Wind Speed** defined in the **Visual Environment**. |
```
Comment (pierre-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/Override-Volumetric-Clouds.md:110]: Typo: "HorizontalWind" should be "Horizontal Wind"

**Review by pierre-unity — CHANGES_REQUESTED** [review]
Only minor typo in the doc.

Tested the actual feature and working well.

**Review by Vic-Cooper — APPROVED** [review]
lgtm!

**Comment by anisunity** [comment]
errors in the tests have nothing to do with clouds @JulienIgnace-Unity

**Review by pierre-unity — APPROVED** [review]
All good, working well and typo fixed.

**Pull request merged**